<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>1101</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Кучка 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pile2Label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>100</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Кучка 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pile3Label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>100</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Кучка 4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pile4Label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>pile4Label</zorder>
      <zorder>label_11</zorder>
     </widget>
     <widget class="QSpinBox" name="pileSpinBox">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>250</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>4</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="numItemsSpinBox">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>310</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QPushButton" name="humanMoveButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>380</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Сделать Ход</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Кучка 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pile1Label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>pile1Label</zorder>
      <zorder>label_6</zorder>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>250</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Номер кучи</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>310</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Колл-во камней</string>
      </property>
     </widget>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget_2</zorder>
     <zorder>layoutWidget_3</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>label_5</zorder>
     <zorder>label_8</zorder>
     <zorder>numItemsSpinBox</zorder>
     <zorder>pileSpinBox</zorder>
     <zorder>humanMoveButton</zorder>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>861</width>
        <height>531</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Pict/msg268862666-114708.000001.jpg</pixmap>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>90</y>
        <width>231</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="startGameButton">
         <property name="text">
          <string>Начать игру</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="infoButton">
         <property name="text">
          <string>Информация</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="text">
          <string>Выход</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QComboBox" name="difficultyComboBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>230</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maxVisibleItems">
       <number>10</number>
      </property>
      <property name="minimumContentsLength">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="startGamePlayButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>281</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Старт</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Сложность</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Колличество Кучек (3-4)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="numPilesSpinBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="maximum">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>312</x>
        <y>9</y>
        <width>781</width>
        <height>531</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Pict/maxresdef.jpg</pixmap>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>301</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="twoPlayerModeRadioButton">
         <property name="text">
          <string>Два игрока</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="humanFirstRadioButton">
         <property name="text">
          <string>Игрок ходит первым</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="aiFirstRadioButton">
         <property name="text">
          <string>Компьютер ходит первым</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1178</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
